Client/src/Components/EventCreation/FetchAltText.jsx

```diff
@@ -8,16 +8,16 @@ function FetchAltText({ eventId }) {
     useEffect(() => {
         const timer = setTimeout(() => {
             setShowButton(true);
-        }, 10000);
+        }, 12000);
```

Comment:
This could be better to rely on server confirmation rather than a fixed time delay to avoid having a bad user experience if the button does not show within the time frame or if those 12 seconds are too long for the user.

Reply:
This will be taken out when I refactor in 2.0.